packages/react-components/src/permissions/PermissionsProvider.test.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { mountWithPermissions } from '../components/utils/testUtils';
+import { CameraButton, ControlBarButton } from '../components';
+import { _getPermissions, _Permissions } from './PermissionsProvider';
+import { registerIcons } from '@fluentui/react';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const mockPresenterPermissions: _Permissions = {
+  cameraButton: true,
+  microphoneButton: true,
+  screenShare: true,
+  participantList: true,
+  removeParticipantButton: true
+};
+
+const mockConsumerPermissions: _Permissions = {
+  cameraButton: false,
+  microphoneButton: false,
+  screenShare: false,
+  participantList: false,
+  removeParticipantButton: false
+};
+
+const mockAttendeePermissions: _Permissions = {
+  cameraButton: true,
+  microphoneButton: true,
+  screenShare: false,
+  participantList: true,
+  removeParticipantButton: false
+};
+
+describe('PermissionProvider tests for camera by passing mock permissions', () => {
+  beforeEach(() => {
+    registerIcons({
+      icons: {
+        controlbuttoncameraoff: <></>,
+        chevrondown: <></>,
+        controlbuttoncameraon: <></>
+      }
+    });
```

Comment:
What do we do if we don't registerIcons? Will it throw erros?

Reply:
Yes, it throws warnings and even adds delay to the tests
<img width="1227" alt="Screen Shot 2022-08-05 at 3 44 00 PM" src="https://user-images.githubusercontent.com/97124699/183220610-74de4996-db90-4de4-ba93-809584f93804.png">
.